### ISSUE TITLE
research(negation): measure the forward-pass depth cost of negation via layerwise logit lens

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#4446]
## Context

The negation-operator thesis holds that the model's global workspace runs best on *positive* state: asking it to represent `not-X` forces it to first construct `X`, then suppress or invert it across depth, spending forward-pass compute the operator ultimately pays for. Before we can build an operator that reduces that tax, we need to measure it. This issue quantifies the *depth cost of negation*: how many layers / how much compute the in-kernel model burns emulating `not-X` versus directly handling `X`. That number is the baseline the negation operator must beat, and it anchors every downstream framing decision.

## Current state

`internal/model/logitlens.go` already ships the exact probe we need: `Model.LayerLogits(act, pos)` projects the residual stream after every layer through the model's own final-norm + LM head, returning per-layer vocab logits, and `TopK` ranks them into softmax probabilities. `cmd/lensviz` consumes it as a per-layer next-token debugger. What does **not** exist is any experiment that *contrasts* negated vs affirmative prompts through this lens, nor any metric summarizing the layer at which the correct answer crystallizes. No negation-tax number exists today.

## Approach

Build a small A/B probe under `internal/model` (test-driven, no new hot-path code): construct matched prompt pairs — an affirmative `X` and its negation `not-X` targeting the same ground-truth token — run one `Forward` each, call `LayerLogits` at the answer position, and compute the *crystallization depth*: the earliest layer whose `TopK[0]` is the correct token and stays correct. The negation tax is `depth(not-X) - depth(X)`, aggregated over a fixture set. Emit the per-pair depths and the mean/median delta as the witness.

## Definition of Done

- [ ] A negation/affirmation fixture set (matched target tokens) lands under `internal/model/testdata`.
- [ ] A helper computes per-layer crystallization depth from `LayerLogits` output.
- [ ] `go test ./internal/model -run TestNegationDepthCost` passes and asserts the tax is measured (delta computed, non-degenerate).
- [ ] The test emits per-pair and aggregate depth deltas to a captured artifact.
- [ ] Honest reporting: if the tax is near zero or negative on this model, that is recorded, not hidden.

## Witness

Captured `go test ./internal/model -run TestNegationDepthCost -v` output showing per-pair crystallization depths for `X` vs `not-X` and the aggregate mean/median layer-delta (the tax).

## Lane

negframe

## Path hints

- internal/model
- internal/model/logitlens.go
- internal/model/testdata

Part of epic #4402.

**Comment by anthony-chaudhary** [comment]
Closed after independent trunk witness: commit 2c98435b0 is present in current ancestry and issue_closure_audit classifies #4446 as OPEN_WITNESSED.

**Comment by anthony-chaudhary** [comment]
Shipped in `2c98435b0441c24123055ceeaff5afd9b3a017fa` (`internal/model` module).

Witness (`go test ./internal/model -run 'TestNegationDepthCost|TestCrystallizationDepth|ExampleCrystallizationDepth' -count=1 -v` in a clean `git archive HEAD` checkout with the owned paths overlaid):

```text
pair=door_open affirmative_depth=1 negated_depth=3 delta=+2
pair=lamp_on affirmative_depth=2 negated_depth=3 delta=+1
pair=route_safe affirmative_depth=3 negated_depth=3 delta=+0
pair=account_active affirmative_depth=3 negated_depth=2 delta=-1
pair=switch_enabled affirmative_depth=0 negated_depth=2 delta=+2
aggregate pairs=5 mean_delta=+0.800 median_delta=+1.000 positive=3 zero=1 negative=1
PASS
```

`go vet ./internal/model` also passed in that isolated checkout. The committed fixture explicitly labels this as a deterministic `LayerLogits` seam witness, not a weight-backed language-model estimate; zero and negative rows are retained rather than hidden. GitHub commit API independently confirms exactly the helper, test, and fixture files, and `origin/main` contains the commit.
